CourierMQTT/MQTT/Connection/MQTT-Client-Framework/MQTTClientFrameworkSessionManager.swift

```diff
@@ -144,7 +144,7 @@ class MQTTClientFrameworkSessionManager: NSObject, IMQTTClientFrameworkSessionMa
         printDebug("MQTT - COURIER: Client Session Manager connect to: \(host)")
         self.connectOptions = connectOptions
         let shouldReconnect = self.session != nil
-        let isTls = port == 443
+        let isTls = port == 443 || securityPolicy != nil
```

Comment:
we can simply use this logic for consistency from above change

```swift
let isTls = connectOptions.scheme == "ssl"
```

Reply:
Done mas. In addition, I add the useSSLWithTLS to check the condition. Thank you